scripts/train.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+
+import hydra
+import lightning.pytorch
+import torch.utils.data as data
+from omegaconf import DictConfig, OmegaConf
+
+from openretina.data_io.cyclers import LongCycler, ShortCycler
+
+log = logging.getLogger(__name__)
+
+
+@hydra.main(version_base="1.3", config_path="../configs", config_name="hoefling_2024_core_readout_high_res")
+def main(cfg: DictConfig) -> None:
+    log.info("Logging full config:")
+    log.info(OmegaConf.to_yaml(cfg))
+
+    ### Import data
+    movies_dict = hydra.utils.call(cfg.data_io.stimuli)
+    neuron_data_dict = hydra.utils.call(cfg.data_io.responses)
+
+    if cfg.check_matches:
```

Comment:
```suggestion
    if cfg.check_matches_between_stimuli_and_responses:
```
Be more verbose (not sure if my proposal is perfect, though)